Packs/Core/Playbooks/playbook-T1059_-_Command_and_Scripting_Interpreter.yml

```diff
@@ -1189,8 +1189,8 @@ tasks:
     scriptarguments:
       process_command_line:
         complex:
-          root: alert
-          accessor: targetprocesscmd
+          root: commandline
+          accessor: decoded
```

Comment:
the decoded will exist only if we found base64 in the command line. you should take of values (decoded and original).